plugins/rhacm/src/components/ClusterStatusCard/ClusterStatusCard.tsx

```diff
@@ -12,26 +12,25 @@ import { useCluster } from '../ClusterContext';
 
 export const ClusterStatusCard = (): any => {
   const { data, loading, error } = useCluster();
-  let reason = data.status.reason;
+  let reason: string;
+  let down = false;
+  let available = false;
 
   if (error) {
-    data.status.reason = 'Unavailable';
+    reason = 'Unavailable';
   } else if (loading) {
-    /*
-    Can't do data.status.reason = 'Loading' since the value
-    will not be overwritten by the state update of the cluster (dunno why)
-    */
     reason = 'Loading';
+  } else {
+    reason = data.status.reason
+    available = data.status.available
+    down = !available && reason === 'Cluster is down'
```

Comment:
Why does have `down` state additional condition? What other state we support for `!available` when `error` is falsy? I don't understand this bit.

Reply:
My thinking behind it was that if `available` is false it doesn't necessarily mean the cluster is down. It could be that ACM is down but the queried cluster is up.

Reply:
When I think about it again maybe it would be better to have two bool values in the status field. 
```json
{
  "status" {
	  "up": true
	  "down": false
      "reason": "Cluster is up"
  }
}
```
Then we don't have to rely on a string value if a cluster is really up or down. WDYT?

Reply:
Never mind I found a better solution I think.